castled-notifications/src/main/res/layout/castled_inapp_fs_html.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<io.castled.android.notifications.inapp.views.InAppWebViewLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/castled_inapp_modal_root"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:padding="0dp">
+
+    <androidx.cardview.widget.CardView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <io.castled.android.notifications.inapp.views.InAppConstrainedLayout
+            android:id="@+id/castled_inapp_html_container"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:padding="0.0dp">
+
+            <WebView
+                android:id="@+id/castled_inapp_webview"
+                style="@style/Castled.InApp.Modal.Layout"
```

Comment:
ideally style name in this case should be `Castled.InApp.WebView.FullScreen`. Also better to define a separate style to avoid any side effects in future